service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/create/notebook/CreateAiNotebookInstanceStep.java

```diff
@@ -95,15 +95,7 @@ public StepResult doStep(FlightContext flightContext)
         return new StepResult(StepStatus.STEP_RESULT_FAILURE_RETRY, e);
       }
 
-      creationOperation =
-          OperationUtils.pollUntilComplete(
-              creationOperation, Duration.ofSeconds(20), Duration.ofMinutes(12));
-      if (creationOperation.getOperation().getError() != null) {
-        throw new RetryException(
-            String.format(
-                "Error creating notebook instance %s. %s",
-                instanceName.formatName(), creationOperation.getOperation().getError()));
-      }
+      GcpUtils.pollUntilSuccess(creationOperation, Duration.ofSeconds(20), Duration.ofMinutes(12));
```

Comment:
maybe make constants for the durations since they appear twice

Reply:
Create and delete are independent operations, so we might actually want to vary them independently. And these numbers don't matter that much as they're overly conservative.